query-execution-plan.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: Understand the Query Execution Plan
 summary: Learn about the execution plan information returned by the `EXPLAIN` statement in TiDB.
-aliases: ['/docs/stable/query-execution-plan/','/docs/v4.0/query-execution-plan/','/docs/stable/reference/performance/understanding-the-query-execution-plan/']
+aliases: ['/docs/stable/query-execution-plan/','/docs/stable/reference/performance/understanding-the-query-execution-plan/','/docs/stable/index-merge/','/docs/stable/reference/performance/index-merge/','/tidb/stable/index-merge']
```

Comment:
There should 7 aliases: @ran-huang @ireneontheway 

['/docs/stable/query-execution-plan/','/docs/v4.0/query-execution-plan/','/docs/stable/reference/performance/understanding-the-query-execution-plan/','/docs/stable/index-merge/','/docs/v4.0/index-merge/','/docs/stable/reference/performance/index-merge/','/tidb/stable/index-merge']

Reply:
fixed in https://github.com/pingcap/docs/pull/3429